ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -478,6 +493,47 @@ internal class CustomerCenterViewModelImpl(
         }
     }
 
+    override fun refreshStateIfLocaleChanged() {
+        val currentLocaleList = getCurrentLocaleList()
+        if (_lastLocaleList.value != currentLocaleList) {
+            _lastLocaleList.value = currentLocaleList
+        }
+    }
+
+    override fun refreshStateIfColorsChanged(colorScheme: ColorScheme, isDark: Boolean) {
+        var shouldRefresh = false
+
+        if (isDarkMode != isDark) {
+            isDarkMode = isDark
+            shouldRefresh = true
+        }
+        if (_colorScheme.value != colorScheme) {
+            _colorScheme.value = colorScheme
+            shouldRefresh = true
+        }
+
+        if (shouldRefresh) {
+            _state.update { it }
+        }
```

Comment:
I wanted to trigger a refresh, assuming this is how darkMode refresh works.

Reply:
Hmm I believe this updates the state with itself, so it wouldn't actually do any change. We would need to add some property to the state that actually needs to be updated I think? (which might come in a separate PR). Otherwise this is not needed I believe.